.github/workflows/kafka_demo_test.yml

```diff
@@ -53,9 +53,9 @@ jobs:
           echo "Extracting Environment Variables"
           echo "Triggering event is ${{ github.event_name }}"
           if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-            echo "key-release-image=mcr.microsoft.com/aci/skr:2.7" >> $GITHUB_OUTPUT
-            echo "consumer-image=mcr.microsoft.com/acc/samples/kafka/consumer:2.0" >> $GITHUB_OUTPUT
-            echo "producer-image=mcr.microsoft.com/acc/samples/kafka/producer:2.0" >> $GITHUB_OUTPUT
+            echo "key-release-image=fishersnpregistry.azurecr.io/skrdb:1" >> $GITHUB_OUTPUT
+            echo "consumer-image=fishersnpregistry.azurecr.io/kafkaminorimprovement/consumer:1" >> $GITHUB_OUTPUT
+            echo "producer-image=fishersnpregistry.azurecr.io/kafkaminorimprovement/producer:1" >> $GITHUB_OUTPUT
```

Comment:
These need to get fixed to the appropriate registries.

Reply:
It would also be great to just move the registry into a variable instead of hard-coded.